test/common/upstream/health_checker_impl_test.cc

```diff
@@ -495,7 +495,7 @@ TEST_F(HttpHealthCheckerImplTest, SuccessServiceCheckWithCustomHostValue) {
         EXPECT_TRUE(headers.Path());
         EXPECT_NE(nullptr, headers.Host());
         EXPECT_NE(nullptr, headers.Path());
-        EXPECT_EQ(headers.Host()->value().c_str(), std::string(host));
+        EXPECT_EQ(headers.Host()->value().c_str(), host);
         EXPECT_EQ(headers.Path()->value().c_str(), std::string("/healthcheck"));
```

Comment:
Why not this line as well?

Reply:
Ah, OK. BTW @htuch do you think it is worth adding more tests as V2 YAML in this test file?

Reply:
Generally, moving as much as possible to v2 YAML will be a good thing. I think the best model to do it is organically, e.g. as we go and modify or add new tests, but any cleanup is always appreciated.